src/gradleinttest.sh

```diff
@@ -21,12 +21,15 @@ cleaniampolicy () {
 }
 
 gradleinttest () {
-  if [ -f env_vars ] && [[ -n "${IT_JADE_API_URL}" ]] && [[ "${test_to_run}" == "testIntegration" ]]; then
-    echo "Getting GCR tags and IT_JADE_API_URL for integration test"
+  if [ -f env_vars ] && [[ -n "${IT_JADE_API_URL}" ]]; then
+    echo "Getting GCR tags and IT_JADE_API_URL for connected or integration test"
     eval $(cat env_vars)
-  else
-    echo "Skipping importing environment vars for gradleinttest"
   fi
+
+  # determine id of integration project
+  namespace_number=$(echo ${NAMESPACEINUSE} | sed 's/integration-//g')
+  google_data_project="broad-jade-int-${namespace_number}-data"
```

Comment:
Can we add an echo here w/ the data project to help w/ debugging?

Reply:
Good idea. Done.